webapp/src/ts/services/cht-script-api.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Injectable, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+import * as chtScriptApiFactory from '@medic/cht-script-api';
+
+import { UserSettingsService } from '@mm-services/user-settings.service';
+import { SettingsService } from '@mm-services/settings.service';
+import { ChangesService } from '@mm-services/changes.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CHTScriptApiService implements OnDestroy {
+  subscriptions: Subscription = new Subscription();
+
+  constructor(
+    private userSettingsService: UserSettingsService,
+    private settingsService: SettingsService,
+    private changesService: ChangesService
+  ) { }
+
+  init() {
+    this.watchChanges();
+
+    return this
+      .getSettings()
+      .then(() => this.getUserSettingsDoc());
+  }
+
+  ngOnDestroy(): void {
```

Comment:
I don't think this is ever destroyed.

Reply:
Yeah, since this service is injected in the root, so not much benefit from unsubscribing observables. 
I'm wondering about the tests like e2e, it should be fine 🤔 but do you think this could be a memory leak problem for e2e?

Reply:
Removed, should be fine